### ISSUE TITLE
A11y_PowerShell Core_PowerShell Editor Services_Installing Powershell in Vscode_Info and Relationships: Focus is landing on Non Interactive elements such as " Powershell" unlabelled image

### ISSUE THREAD

**Issue by Latha2564** [microsoft/vscode#135448]
GitHubTags:#A11y_VSCode;#A11ySev2;#A11yTCS;#A11yMAS;#DesktopApp;#WCAG1.3.1;#InfoandRelationships;#MAC;#VoiceOver;

### Actual Result:​
Focus is landing on Non Interactive elements such as " PowerShell" unlabeled image

### Expected Result:​
Focus should be landing on Interactive elements and control only.

### Environment Details:​
Application Name: PowerShell Editor Services

### Repro Steps:
Download VS Code Space Insiders 
Open VS Code Space insiders 
Tab till Extension in the left navigation and press enter
Tab till Search Extensions in Marketplace and type PowerShell and press enter
Tab till "PowerShell" unlabeled image and check whether focus is landing on Non Interactive elements or not.

### User Impact:​
Screen Readers & Keyboard users will have difficulty in accessing such non interactive controls or elements Focus is landing on Non Interactive elements such as " PowerShell" unlabelled image

### Attachments:
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-lkondreddy_microsoft_com/EZl5cdL0fsJOlWETRFWT0hQBeW_Azb9bSZazPLkUx8bvsA?e=9wG0jI

**Comment by isidorn** [comment]
Accessibility issue for extensions should be filled in the repository of the extension. Once that is done the extension author should investigate and file an issue in VS Code using proper description of our components. Thanks!

Also this issue has not video.
